.cruft.json

```diff
@@ -0,0 +1,16 @@
+{
+    "template": "https://github.com/HERMES-SOC/hermes_instrument.git",
+    "commit": "35cde43220942734867e5207ff69464435120c1a",
+    "checkout": null,
+    "skip": [
+      "README.rst/",
+      "docs",
```

Comment:
we want to be able to make changes to conf.py so not all files under docs should be ignored. we also want to be able to make updates to the doc template. See PR #10

Reply:
Yeah, I've removed it, it only makes updates based on whatever commits it is based on. I thought it would capture all the diffs up to the commit specified. But it turns out it only checks for diffs after the set commit hash. So this should be fine.